grub/pkg.yaml

```diff
@@ -0,0 +1,32 @@
+name: grub
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - stage: util-linux
+steps:
+  - sources:
+      - url: https://ftp.gnu.org/gnu/grub/grub-2.04.tar.xz
+        destination: grub.tar.xz
+        sha256: e5292496995ad42dabe843a0192cf2a2c502e7ffcc7479398232b10a472df77d
+        sha512: 9c15c42d0cf5d61446b752194e3b628bb04be0fe6ea0240ab62b3d753784712744846e1f7c3651d8e0968d22012e6d713c38c44936d4004ded3ca4d4007babbb
+    prepare:
+      - |
+        tar -xJf grub.tar.xz --strip-components=1
+
+        ./configure --prefix=/usr \
+                    --sbindir=/sbin \
+                    --sysconfdir=/etc \
+                    --disable-efiemu \
+                    --disable-werror
+    build:
+      - |
+        make -j $(nproc)
+    install:
+      - |
+        make install DESTDIR=/rootfs
+        mkdir -p /rootfs/usr/share/bash-completion/completions
```

Comment:
do we really need bash completion?

Reply:
Not in Talos. But this package should be able to stand on its' own. I envision that other distros might be able to help us maintain these packages in the future. Moving them to `/usr/share` ensures that they get removed by the Talos build: https://github.com/talos-systems/talos/blob/master/hack/cleanup.sh#L31.